package.json

```diff
@@ -184,16 +177,15 @@
     "webpack": "^5.75.0",
     "zx": "^7.2.0"
   },
-  "dependencies": {
-  },
+  "dependencies": {},
   "peerDependencies": {
-    "monaco-editor": "~0.34.0",
-    "vscode-oniguruma": "^1.6.2",
-    "vscode-textmate": "^7.0.1"
+    "monaco-editor": "~0.36.1",
+    "vscode-oniguruma": "^1.7.0",
```

Comment:
This is not blocking, btw

Reply:
monaco-editor needs to be fixed very precisely because any change can be a breaking change for us as we import internal apis

oniguruma and textmate aren't supposed to introduce any breaking change in patch version, so I guess it's fine?

Reply:
Alright!